registry/client.go

```diff
@@ -1061,17 +1064,23 @@ func (c *client) GetActivePoolStake(blockNumber, poolID *big.Int) (*big.Int, err
 	return big.NewInt(0).Add(poolStake, delegatorPoolStake), nil
 }
 
-func (c *client) TotalScannersRegistered(blockNumber, poolID *big.Int) (*big.Int, error) {
+func (c *client) GetAllocatedStakePerManaged(blockNumber, poolID *big.Int) (*big.Int, error) {
 	contracts := c.Contracts()
-	if contracts.ScannerPoolReg == nil {
```

Comment:
I think need an `if contracts.StakeAllocator == nil` here. The stake allocator does not exist in production right now and it will be detected and initialized when the new contracts are deployed to production.